bin/templates/platform_www/cdv-electron-main.js

```diff
@@ -62,10 +89,27 @@ function createWindow () {
     });
 }
 
+function configureProtocol () {
+    protocol.registerFileProtocol(scheme, (request, _cb) => {
+        const url = request.url.substr(basePath.length + 1);
+        _cb({ path: path.normalize(`${__dirname}/${url}`) });
+    }, (error) => {
+        if (error) console.error('Failed to register protocol');
+    });
+
+    protocol.interceptFileProtocol('file', (_, _cb) => { _cb(null); });
```

Comment:
Same as above:

any reason for prefixing `cb`? Normally, I'll do this only if it's unused, but it's called.